discovery/moby/dockerswarm.go

```diff
@@ -39,19 +39,19 @@ const (
 var userAgent = fmt.Sprintf("Prometheus/%s", version.Version)
 
 // DefaultSDConfig is the default Docker Swarm SD configuration.
-var DefaultSDConfig = SDConfig{
+var DefaultSDConfig = DockerSwarmSDConfig{
```

Comment:
Should this also be renamed to `DefaultDockerSwarmSDConfig`?

---

Am I right in assuming this field (`DefaultSDConfig`) is public for use in documentation, because I do not see it used outside of the package?

Reply:
Many other packages import this and might change this.

Reply:
True, I somehow missed that it can also be used by external packages...

Reply:
That said, this PR renames the package, so wouldn't every consumer need to change this anyway?

Reply:
Yes, that is not a concern.

Reply:
I was telling why it is public, not that we should not rename it :)